Storage/src/StorageObject.php

```diff
@@ -146,7 +146,7 @@ public function acl()
     public function exists(array $options = [])
     {
         try {
-            $this->connection->getObject($this->identity + $options + ['fields' => 'name']);
+            $this->connection->getObject($options + array_filter($this->identity) + ['fields' => 'name']);
```

Comment:
Why is this change needed here?

Reply:
Is it okay to change the order just to make sure it works as before:
```
$this->connection->getObject(array_filter($this->identity) + $options + ['fields' => 'name']);
```

Reply:
I think it might've gotten in by mistake. This is a great catch. Fixed it.